src/components/program/styles/_ProgramCourses.scss

```diff
@@ -7,3 +7,7 @@
     }
   }
 }
+
+.calendar-icon {
```

Comment:
nit: Rather than defining a custom class name for adding `margin-bottom`, I'd recommend using the CSS utility classes related to spacing. In this case, `mb-1`: https://paragon-openedx.netlify.app/foundations/spacing/

Rationale: more consistent spacing throughout the UI. If/when the standard spacing values change for the `@edx/brand-edx.org` theme, using the CSS utility classes or SCSS variables from Paragon will ensure future design changes to Paragon propagate to consuming applications 😄

Reply:
Thank you for sharing the paragon source! I didn't know that was available.